backend/cmdlineoptions.go

```diff
@@ -50,6 +65,43 @@ func init() {
 		VolumePctCLIArg = v
 		return err
 	})
+
+	if term.IsTerminal(int(os.Stdin.Fd())) {
+		flag.Func("play-album-by-id", "start playing the album with the given ID (can also be passed from standard input)", func(s string) error {
+			PlayAlbumCLIArg = s
+			return nil
+		})
+		flag.Func("play-playlist-by-id", "start playing the playlist with the given ID (can also be passed from standard input)", func(s string) error {
+			PlayPlaylistCLIArg = s
+			return nil
+		})
+		flag.Func("play-track-by-id", "start playing the track with the given ID (can also be passed from standard input)", func(s string) error {
+			PlayTrackCLIArg = s
+			return nil
+		})
+	} else {
+		FlagPlayAlbum = flag.Bool("play-album-by-id", false, "")
+		FlagPlayPlaylist = flag.Bool("play-playlist-by-id", false, "")
+		FlagPlayTrack = flag.Bool("play-track-by-id", false, "")
+	}
+	flag.Func("first-track", "start playing from given track (positive integer, to be used with either -play-album or -play-playlist)", func(s string) error {
+		v, err := strconv.Atoi(s)
+		FirstTrackCLIArg = v
+		return err
+	})
+
+	flag.Func("search-album", "search album", func(s string) error {
```

Comment:
Can we add more descriptive info for these search flags, something like "search albums by name, return results as JSON"